modules/unsupported/jdbc-ng/jdbc-hana/src/main/java/org/geotools/data/hana/converter/BlobConverterFactory.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2018, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.data.hana.converter;
+
+import java.sql.Blob;
+import org.geotools.factory.Hints;
+import org.geotools.util.Converter;
+import org.geotools.util.ConverterFactory;
+
+/**
+ * Converter factory for converting all classes implementing java.sql.Blob to byte[].
+ *
+ * @author Stefan Uhrig, SAP SE
+ */
+public class BlobConverterFactory implements ConverterFactory {
```

Comment:
Not something that needs to be changed, just an heads up. This converter (and the other two) are targeting native Java types, i.e. nothing specific to SAP HANA, why not contributing them to the ``org.geotools.util`` GeoTools package ? 

Otherwise there is a risk that someone will contribute a converter that does the same conversion (Blob -> byte[]) and the behavior will may be different depending on the converter loaded first. For example, ``org.geotools.data.teradata`` already implements a Blob -> byte[] converter.

Reply:
I propose I do that in a separate pull request (for `Blob` and `Clob`) and remove the converter from `org.geotools.data.teradata` then as well.

Reply:
Sounds good to me, thank you.